lib/FileUploaderField/tests/interactor.js

```diff
@@ -0,0 +1,22 @@
+import {
+  clickable,
+  interactor,
+  isPresent,
+  text,
+} from '@bigtest/interactor';
+
+@interactor class FileUploaderFieldInteractor {
+  static defaultScope = '[data-test-document-field-file]';
+
+  hasFilename = isPresent('[data-test-fuf-name]');
+  hasUploaded = isPresent('[data-test-fuf-uploaded]');
+  hasDeleteBtn = isPresent('[data-test-fuf-delete]');
+
+  filename = text('[data-test-fuf-name]');
+  uploaded = text('[data-test-fuf-uploaded]');
+  delete = clickable('[data-test-fuf-delete]');
```

Comment:
[`delete` is a reserved keyword in JS.](https://ecma-international.org/ecma-262/6.0/#sec-reserved-words) I guess this didn't throw any errors beforehand but we should take the chance to give it a better name. (Pretty sure I wrote this line originally. 🤦‍♂ )

Reply:
This line https://github.com/folio-org/stripes-erm-components/blob/e0119dd000d98173c4ff3d119f6cd86116e4b38c/lib/DocumentsFieldArray/tests/DocumentsFieldArray-test.js#L89 should take care of it? shouldn't it? unless you are referring to something else here?